LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2021, QIIME 2 development team.
+Copyright (c) 2022-2022, QIIME 2 development team.
```

Comment:
2022-2022 looks weird...

why not 2021-2022 or just 2022? (I realize this is not your problem, it's issues with q2lint and us not yet recognizing the new year, so we can fix if this is a hassle)

Reply:
I hope this works with the linting